Support set `location`

wangxiyuan opened the pull request:
Add a new parameter to support set the location where the file
will be saved.
e.g.: -service keystone -version queens -location E:\123\456\
TODO: the usage guide should be updated.

Yikun commented on bpget/bp_get.go:
```diff
@@ -56,18 +56,43 @@ func GetBlueprint(service string, version string) []*BluePrint {
+
+		for i := 1; i < deep; i++ {
+			folder = folder + "\\" + folders[i]
+			os.Mkdir(folder, os.ModeDir)
```
try to use [os.MkdirAll](https://golang.org/pkg/os/#MkdirAll), it seems more convenient to create directory recursively, and also more friendly to mkdir in different OS.

Yikun commented on bpget/bp_get.go:
```diff
@@ -56,18 +56,43 @@ func GetBlueprint(service string, version string) []*BluePrint {
 func main() {
 	servicePtr := flag.String("service", "nova", "The service name of OpenStack (nova, cinder...)")
 	versionPtr := flag.String("version", "queens", "The version of service (pike, queens...)")
+	locationPtr := flag.String("location", "", "The location of the output file.")
```
It would be good to add some location example in here, like:
> The location of the output file. (c:/folder, ../folder, ./folder)

wangxiyuan committed:
Support set `location`

Add a new parameter called `location` to indecate where the file
will be saved.
